Libraries/ConfigParser/ConfigParser.Tests/ConfigParserTests.cpp

```diff
@@ -57,11 +57,11 @@ namespace ConfigParserTests
 			Assert::AreEqual("test:test:1234", injectedWebRTCInstance->turn_server.uri.c_str());
 			Assert::AreEqual("test", injectedWebRTCInstance->turn_server.username.c_str());
 			Assert::AreEqual("test", injectedWebRTCInstance->turn_server.password.c_str());
-			Assert::AreEqual("test", injectedWebRTCInstance->server.c_str());
+			Assert::AreEqual("test", injectedWebRTCInstance->server_uri.c_str());
```

Comment:
can we add a new test that validates this fallback behavior?

Reply:
Done! I had to change some of the code in config_parser and remove the hardcoded stuff, but there are now functional unit tests included.